### DISCUSSION TITLE
Desperate with the latest updates

### DISCUSSION THREAD

**Discussion by JoseAltea** [FrSkyRC/ETHOS-Feedback-Community#4124] [Q&A]
Hello, good morning. Since the last updates were made for me, it has been a total trauma. This had never happened to me, I already communicated it days ago, that having to adjust the curves of the mixtures was very complicated since you had to continuously change the screen for each parameter, before any modification was done **EVERYTHING** on the same screen and without stopping seeing the curves and thus being able to see its evolution. This was told to me that it would be solved in the next updates, which I do not see that it has been solved, it remains the same, at least on my transmitter a **TANDEM XE**. I also have problems with the management of the images of the models, they look perfectly in. The SD com on the RADIO, but when I access the images to be able to change them while I'm editing the model you can't see the one I want, it's not, it's as if I read them from another place that I can't access. As for the sounds that I want to use with the special functions, the same thing happens, I attach them to the root directory of audios but when I want to configure a special function and associate it with an audio, I can't find any of those in the root, the truth is that I don't understand anything. I don't know if I'm doing it wrong or if there's an error in the updates, I don't know if the same thing happens to other users of this transmitter. I hope you take this into account and can solve these problems for me.

**Comment by GiacomoGaleotti** [comment]
did you put the audio files in the right new folder?
i can't answere for other issue but i had problem with audio files because root is changed from 1.4...

**Comment by JoseAltea** [comment]
Good morning Giacomo. Thank you for the information, both the Audio and the images have already been solved. As for the configuration of the mixes, I commented on it a long time ago and they told me that they would solve it since it is very uncomfortable to have to switch from one screen to another to have to modify any parameter. 
Thank you again and best regards.
Jose

**Comment by bsongis** [comment]
Have you checked if the issue you are mentionning is marked as fixed? And if fixed, in which version it is (will be) included?

**Comment by JoseAltea** [comment]
Hello, good morning. I put the comment on March 30, Problems latest versions #3749 The version was 1.5.1. In this latest version, 1.5.9, it has not changed. When you have to add a curve or some action, change the screen to select the desired curve or action, this is not bad, the problem is that when I want to edit the value/weight of the action or curve, it sends me out of the screen to enter the new value, what I suggest is that once the new curve or action is created and we want to modify the value does not change the screen and it can be modified from the same line in which the value appears.

For example, if I want to change the value from 65% to 70%, I have to select the line I want to modify.

￼

Select Edit.

￼

Look for the value.

￼

Select the value.

￼

Edit the value.

￼

And give the RTN 3 times to go back to the page and be able to see the result.

I hope I have explained myself well. Thank you and regards.
Jose.

**Comment by JoseAltea** [comment]
Hello, good morning. I put the comment on March 30, Problems latest versions [#3749](https://github.com/FrSkyRC/ETHOS-Feedback-Community/issues/3749) The version was 1.5.1. In this latest version, 1.5.9, it has not changed. When you have to add a curve or some action, change the screen to select the desired curve or action, this is not bad, the problem is that when I want to edit the value/weight of the action or curve, it sends me out of the screen to enter the new value, what I suggest is that once the new curve or action is created and we want to modify the value does not change the screen and it can be modified from the same line in which the value appears.For example, if I want to change the value from 65% to 70%, I have to select the line I want to modify.

![IMG_5929-min_4_11zon](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/123797860/c2810e63-7259-49c6-ba4e-84f1df8baadc)

Select Edit.

![IMG_5930-min_3_11zon](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/123797860/f7056dcf-ddd0-4d47-b3a2-2272ff2b545e)

Look for the value.

![IMG_5931-min_5_11zon](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/123797860/8406c4b8-c6b5-4b13-856b-5c49ee9849e2)

Select the value.

![IMG_5932-min_1_11zon](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/123797860/1ead88d0-f1a4-426c-a18c-cdbb795f0668)

Edit the value.
![IMG_5933-min_2_11zon](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/123797860/d99f2d22-5b9f-4fe7-8baa-ac6edadfc22f)

And give the RTN 3 times to go back to the page and be able to see the result.
I hope I have explained myself well. Thank you and regards.
Jose.

**Comment by bsongis-frsky** [comment]
Latest 1.5.10 nightly ...

![image](https://github.com/FrSkyRC/ETHOS-Feedback-Community/assets/50834778/fc6d2386-e727-4e0b-b089-4d9a14d9c341)
